sites/authentication/authentication-en.html

```diff
@@ -4,58 +4,61 @@
+	<ul>
+		<li><b>Small screens</b>: total of 12 characters including spaces (no additional descriptive text)</li>
+		<li><b>Large/medium screens</b>: total of 25 characters including spaces (for additional descriptive text if needed, such as “Sign in to [account name]”)</li>
+	</ul>
+	</li>
 </ul>
 
 <h3>HTML Structure - Version 1.0</h3>
 
 <p>Code sample of the include showing the HTML of the component</p>
 <ul>
-	<li><a href="contextual-signin-en.html">Contextual Sign-in button</a></li>
+	<li><a href="contextual-signin-en.html">Contextual Sign in button</a></li>
 	<li><a href="contextual-signin-label-extended-en.html">Contextual Sing-in button with extended label</a></li>
-	<li><a href="contextual-signin-nolabels-en.html">Contextual Sign-in button without custom labels</a></li>
-	<li><a href="contextual-signin-nomenu-en.html">Contextual Sign-in button without the GCWeb Menu</a></li>
+	<li><a href="contextual-signin-nolabels-en.html">Contextual Sign in button without custom labels</a></li>
+	<li><a href="contextual-signin-nomenu-en.html">Contextual Sign in button without the GCWeb Menu</a></li>
 </ul>
 
-<h2>Signed-off / Signed-on</h2>
-
-<p>On page load, the buttons will appear on the top right-hand side of the page, below the horizontal line.</p>
-
-<h3>GCWeb Jekyll Configuration</h3>
+<h3>GCWeb Jekyll configuration</h3>
 
-<p>The <code>auth</code> variable MUST be set along with the <code>type</code> parameter set with either the value "signedoff" or "signedon" in order to initiate the Signed-off or Signed-on pattern.</p>
+<p>The <code>auth</code> variable MUST be set along with the <code>type</code> parameter with the value "contextual" in order to initiate the contextual sign in button pattern.</p>
 
 <p>Parameters to be set:</p>
 <ul>
-	<li><strong>signedoff</strong> set as value to the <code>type</code> parameter will generate the "Register" and "Sign in" buttons to appear on the top right-hand side of the page, below the horizontal line.</li>
-	<li><strong>signedon</strong> set as value to the <code>type</code> parameter will generate the "My account" and "Sign out" buttons to appear on the top right-hand side of the page, below the horizontal line along with the wording "Signed in as [full name]" right underneath.</li>
+	<li><code>link</code> this parameter MUST be set in order to assign a URL to the link button for pointing to the application where users will be redirected.</li>
+	<li><code>label</code> with a maximum lenght of 12 characters allowed including spaces, this parameter MUST be set to generate the text appearing on the button for any viewports from extra-extra-small to extra-large (xxs, xxs, sm, md, lg and xl).</li>
+	<li><code>labelExtended</code> with a maximum lenght of 25 characters allowed including spaces, this parameter is optional but must be set along with the <code>label</code> parameter and if set, it will generate the text appearing on the button for small viewport and up (sm, md, lg and xl).</li>
```

Comment:
```suggestion
	<li><code>labelExtended</code> with a maximum length of large screen allowed including spaces, this parameter is optional but must be set along with the <code>label</code> parameter and if set, it will generate the text appearing on the button for small viewport and up (sm, md, lg and xl).</li>
```